### ISSUE TITLE
Handover `knative.team` GSuite ownership and billing to CNCF

### ISSUE THREAD

**Issue by upodroid** [knative/test-infra#3076]
CNCF needs to take ownership and billing responsibility for knative.team GSuite Subscription. It is currently paid by Google.

/kind cncf-infra

**Comment by kvmware** [comment]
Duplicate of https://github.com/knative/test-infra/issues/2976 this already is mentioned there.

**Comment by upodroid** [comment]
https://support.google.com/nonprofits/answer/3367631?hl=en-GB

**Comment by hh** [comment]
I have access now.

**Comment by hh** [comment]
I'll put this on the LF card next week with help from Taylor W.

**Comment by thisisnotapril** [comment]
We should sync on this. Steering made decisions today about the GSuite account that are going to impact the number of users, etc. 
Steering also kicked off the process with CNCF Servicedesk for this; and I want to be sure we're not duplicating work.

**Comment by hh** [comment]
CNCF Credit Card should for this should be in place early next week.
/cc

**Comment by hh** [comment]
This is done!

**Comment by upodroid** [comment]
Thank you 

/close

**Comment by chizhg** [comment]
Billing hasn't been switched to CNCF billing account yet. All the GCP projects under `knative.team/infra` folder will need to be switched.

```
$ gcloud projects list --filter 'parent.id=1055082993535 AND parent.type=folder' | awk '{print $1 }'
PROJECT_ID
knative-analytics
knative-boskos-01
knative-boskos-02
knative-boskos-03
knative-boskos-04
knative-boskos-05
knative-boskos-06
knative-boskos-07
knative-boskos-08
knative-boskos-09
knative-boskos-10
knative-boskos-100
knative-boskos-11
knative-boskos-12
knative-boskos-13
knative-boskos-14
knative-boskos-15
knative-boskos-16
knative-boskos-17
knative-boskos-18
knative-boskos-19
knative-boskos-20
knative-boskos-21
knative-boskos-22
knative-boskos-23
knative-boskos-24
knative-boskos-25
knative-boskos-26
knative-boskos-27
knative-boskos-28
knative-boskos-29
knative-boskos-30
knative-boskos-31
knative-boskos-32
knative-boskos-33
knative-boskos-34
knative-boskos-35
knative-boskos-36
knative-boskos-37
knative-boskos-38
knative-boskos-39
knative-boskos-40
knative-boskos-41
knative-boskos-42
knative-boskos-43
knative-boskos-44
knative-boskos-45
knative-boskos-46
knative-boskos-47
knative-boskos-48
knative-boskos-49
knative-boskos-50
knative-boskos-51
knative-boskos-52
knative-boskos-53
knative-boskos-54
knative-boskos-55
knative-boskos-56
knative-boskos-57
knative-boskos-58
knative-boskos-59
knative-boskos-60
knative-boskos-61
knative-boskos-62
knative-boskos-63
knative-boskos-64
knative-boskos-65
knative-boskos-66
knative-boskos-67
knative-boskos-68
knative-boskos-69
knative-boskos-70
knative-boskos-71
knative-boskos-72
knative-boskos-73
knative-boskos-74
knative-boskos-75
knative-boskos-76
knative-boskos-77
knative-boskos-78
knative-boskos-79
knative-boskos-80
knative-boskos-81
knative-boskos-82
knative-boskos-83
knative-boskos-84
knative-boskos-85
knative-boskos-86
knative-boskos-87
knative-boskos-88
knative-boskos-89
knative-boskos-90
knative-boskos-91
knative-boskos-92
knative-boskos-93
knative-boskos-94
knative-boskos-95
knative-boskos-96
knative-boskos-97
knative-boskos-98
knative-boskos-99
knative-dns
knative-e2e-dns
knative-gcp
knative-nightly
knative-performance
knative-releases
knative-serving-e2e-tests
knative-tests
```

**Comment by chizhg** [comment]
Once the billing account is ready, you can run below commands to migrate these projects:

```
BILLING_ACCOUNT_ID=[CNCF-billing-account-ID]
PROJECTS=($(gcloud projects list --filter 'parent.id=1055082993535 AND parent.type=folder' --format="csv[no-heading](PROJECT_ID)" | awk '{print $1 }'))
for proj in "${PROJECTS[@]}"
do
  gcloud beta billing projects link "${proj}" --billing-account "${BILLING_ACCOUNT_ID}"
done
```

**Comment by hh** [comment]
We did create a billing account, but there was concern about it not being invoiced based billing. I put in a request to upgrade the new billing account, but it was rejected to to lack of history. (Requires 3+ months).  Today I put in a request to upgrade the existing billing account to be invoiced to the CNCF. We should hear back sometime next week.
